ui/test/nuxeo-collapsible.test.js

```diff
@@ -0,0 +1,158 @@
+    test('Should display nuxeo-collapsible when declaring the element', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible></nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      expect(isElementVisible(nuxeoCollapsible.shadowRoot.querySelector('button'))).to.be.true;
+    });
+
+    test('Should show and hide content when heading is tapped', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible>
+            <span slot="heading">Content</span>
+            <div>Collapsed content</div>
+          </nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      const button = nuxeoCollapsible.shadowRoot.querySelector('button');
+      const content = nuxeoCollapsible.shadowRoot.querySelectorAll('slot')[1].assignedNodes()[2];
+      const ironCollapse = nuxeoCollapsible.shadowRoot.querySelector('iron-collapse');
+      if (isElementVisible(content)) {
+        await waitChanged(ironCollapse, 'aria-hidden');
+      }
+      expect(isElementVisible(content)).to.be.false;
+
+      tap(button);
+      if (!nuxeoCollapsible.expanded) {
+        await waitForAttrMutation(nuxeoCollapsible, 'expanded', true);
+      }
+      expect(isElementVisible(content)).to.be.true;
+      expect(content.innerHTML).to.equal('Collapsed content');
+    });
+  });
+
+  suite('Properties', () => {
+    test('Should display heading when it is set', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible>
+            <span class="heading" slot="heading">Content</span>
+          </nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      const heading = nuxeoCollapsible.shadowRoot.querySelector('slot').assignedElements()[0];
+      expect(isElementVisible(heading)).to.be.true;
+      expect(heading.innerHTML).to.equal('Content');
+    });
+
+    test('Should display chevron when it is set', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible chevron></nuxeo-collapsible>
+        `,
+      );
+      await flush();
+      const chevron = nuxeoCollapsible.shadowRoot.querySelector('iron-icon');
+      expect(isElementVisible(chevron)).to.be.true;
+    });
+
+    test('Should display chevron position defaulted to the end when it is set', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible chevron>
+            <span slot="heading">Content</span>
+          </nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      const heading = nuxeoCollapsible.shadowRoot.querySelector('slot').assignedElements()[0];
+      const chevron = nuxeoCollapsible.shadowRoot.querySelector('iron-icon');
+      expect(isElementVisible(heading)).to.be.true;
+      expect(isElementVisible(chevron)).to.be.true;
+      expect(heading.offsetLeft).to.equal(4);
```

Comment:
Is it `4` because of `--nuxeo-collapsible-heading-padding`, right?

Reply:
Exactly